### DISCUSSION TITLE
Working on Fedora 35 with GNOME 41.6/Wayland

### DISCUSSION THREAD

**Discussion by ARYANTECH123** [Wazzaps/fingerpaint#24] [General]
I had a few perplexing moments when the package did not work the first time I tried it. I think i have it mostly figured out.
Using Fedora 35 Workstation with Gnome 41.6/Wayland. Trying to run `fingerpaint -o out.png` gave this error:
```
Using touchpad: PIXA3854:00 093A:0274 Touchpad
Unexpected touchpad state: "", are you using Gnome?
```

I tracked the error down to `dconf` not being able to read the `send-events` key properly. Changing the status of the key from `dconf-editor` to be editable fixed the issue.

Hope this helps!

**Comment by Wazzaps** [comment]
Huh, that's weird 
Are you using the latest version? (v1.2.6)

**Reply by ARYANTECH123** [reply]
Yes, i am using 1.2.6
Installed from wheel using pip iirc.
Could be something specific to my machine, but it affected a few others running Fedora who have tried.

**Comment by Damianus500** [comment]
I just saw the fingerpaint project and wanted to give it a try. 

My system is 
```
      ///////*767////////////////         OS: Pop!_OS 22.04 LTS x86_64
    //////7676767676*//////////////       Host: HP Laptop 15s-eq0xxx
   /////76767//7676767//////////////      Kernel: 6.0.12-76060006-generic
  /////767676///*76767///////////////     Uptime: 8 mins
 ///////767676///76767.///7676*///////    Packages: 3095 (dpkg), 66 (flatpak)
/////////767676//76767///767676////////   Shell: bash 5.1.16
//////////76767676767////76767/////////   Resolution: 1920x1080
///////////76767676//////7676//////////   DE: GNOME 42.5
////////////,7676,///////767///////////   WM: Mutter
/////////////*7676///////76////////////   WM Theme: Orchis
///////////////7676////////////////////   Theme: Orchis [GTK2/3]
 ///////////////7676///767////////////    Icons: Numix [GTK2/3]
  //////////////////////'////////////     Terminal: tmux
   //////.7676767676767676767,//////      CPU: AMD Ryzen 7 3700U with Radeon Vega Mobile Gfx (8) @ 2.300GHz
    /////767676767676767676767/////       GPU: AMD ATI Radeon Vega Series / Radeon Vega Mobile Series
      ///////////////////////////         Memory: 5101MiB / 13897MiB
```
      
I installed on my Pop Os system using the instructions for debian:

```shell      
apt install xinput python3-pip python3-tk
pip3 install fingerpaint
```

I am using Wayland.

The error message I received is:

```
$ fingerpaint -o dl.png
Using touchpad: ELAN071A:00 04F3:30FD Touchpad
Unexpected touchpad state: "", are you using Gnome?
```

I went to dconf-editor and touchpad/send-events is already "enabled".

What else can I do?

Any help much appreciated.

Thank you.

Damianus

**Reply by Wazzaps** [reply]
Found a workaround, turn off "use a default value" for that key (`/org/gnome/desktop/peripherals/touchpad/send-events`) in `dconf-editor`.
I'll work on a proper fix. Thanks for the report! :)

**Reply by Damianus500** [reply]
Yes. I disable send-events from dconf-editor. But after ONE usage of fingerpaint, my mousepad no longer functions as mouse. I think I have to reboot ot logout. I have to use an external mouse.

Nevertheless, Good effort and useful little app .... there is a use case... even better if it can undo the last few strokes.

Thank you!

Damianus

**Reply by Wazzaps** [reply]
I meant set it to "enabled", just turn off the "use default value" switch, it should work.
I might add Ctrl+z sometime :)

**Comment by Wazzaps** [comment]
This should be fixed by the new release (v1.2.7), if it isn't, reopen #12.
